hybridse/src/node/expr_node.cc

```diff
@@ -918,6 +918,10 @@ ConstNode* ConstNode::ShadowCopy(NodeManager* nm) const {
             LOG(WARNING) << "Fail to copy primary expr of type " << node::DataTypeName(GetDataType());
             return nm->MakeConstNode(GetDataType());
         }
+        
+        default:
+            LOG(ERROR) << "Unsupported Data type";
```

Comment:
Append the current data type to log info
`LOG(ERROR) << "Unsupported Data type " << node::DataTypeName(GetDataType())`